modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/java/JavaModelLoaderDelegate.java

```diff
@@ -168,6 +177,34 @@ public ExtensionDeclarer declare(ExtensionLoadingContext context) {
     return declarer;
   }
 
+  protected void parseExternalLibs(WithAnnotations withAnnotations, DeclaresExternalLibraries declarer) {
+    Optional<ExternalLibs> externalLibs = withAnnotations.getAnnotation(ExternalLibs.class);
```

Comment:
looked at it. Not exactly the same. Besides that depends on classes and I'm using Wasinger's type abstraction